app/controllers/application_controller.rb

```diff
@@ -11,7 +11,9 @@ class ApplicationController < ActionController::Base
   fine_print_require :general_terms_of_use, :privacy_policy, unless: :disable_fine_print
 
   def redirect_to_newflow_if_enabled
-    if request.get? && Settings::Db.store.student_feature_flag && !params[:bpff].present?
+    return false unless request.get?
+
+    if (Settings::Db.store.student_feature_flag || Settings::Db.store.educator_feature_flag) && params[:bpff].blank?
```

Comment:
consider: you could have this as a private method that documents the bpff flag (without needing a comment) and what it's testing), and end the def w/ a ?

Reply:
Welp, this was a bit more involved than I expected it to be. Definitely an excellent suggestion!

Defined these in `ApplicationHelper`:

```
  # Permit legacy flow is a url parameter that, when present, allows us to bypass the newflow feature flag redirects.
  def get_param_permit_legacy_flow
    # bpff in short stands for "bypass feature flag"
    params[:bpff]
  end

  # Permit legacy flow is a url parameter that, when present, allows us to bypass the newflow feature flag redirects.
  def set_param_permit_legacy_flow
    # bpff in short stands for "bypass feature flag" — intentionally made a little obscure.
    params[:bpff] = 9
    params[:bpff].to_h
  end

```